esy-lib/Tarball.re

```diff
@@ -87,6 +87,19 @@ let unpackWithTar = (~stripComponents=?, ~dst, filename) => {
   | Some(stripComponents) =>
     Fs.withTempDir(out => {
       let* () = unpack(out);
+      let normalizedOut = EsyBash.normalizePathForCygwin(Path.show(out));
```

Comment:
Let's place a comment here: "We have observed NPM tarballs to contain root directories with no x bit set, because of which lstat fails. The following lines set the x bit of the root directory, so that the directory contents are searchable". See commit message for more details.